OpenRA.Mods.Common/AI/SupportPowerDecision.cs

```diff
@@ -94,7 +94,7 @@ public int GetAttractiveness(IEnumerable<Actor> actors, Player firedBy)
 		/// <summary>Makes up part of a decision, describing how to evaluate a target.</summary>
 		public class Consideration
 		{
-			public enum DecisionMetric { Health, Value, None }
+			public enum DecisionMetric { Health, Value, BaseBuilding, None }
```

Comment:
Oh, spoonfeeding?! Ok, while you are at it, please spoon feed me the
documentation and design as it appears that they happen to be missing.
Yeah, I understand that this bot is used across different mods that is why
we have configuration in form of yaml. btw, I already mentioned this on the
irc chan and no one mentioned this before. Thanks again for spoon feeding
me your yaml wisdom although it doesn't scratch the issue i'm raising yet.
How will this place more weight in the attractiveness calculation if you
are still using the same value metric?

On Thu, Nov 12, 2015 at 6:38 PM, Zimmermann Gyula <notifications@github.com>
wrote:

> In OpenRA.Mods.Common/AI/SupportPowerDecision.cs
> <https://github.com/OpenRA/OpenRA/pull/9936#discussion_r44680079>:
>
> > @@ -94,7 +94,7 @@ public int GetAttractiveness(IEnumerable<Actor> actors, Player firedBy)
> >  		/// <summary>Makes up part of a decision, describing how to evaluate a target.</summary>
> >  		public class Consideration
> >  		{
> > -			public enum DecisionMetric { Health, Value, None }
> > +			public enum DecisionMetric { Health, Value, BaseBuilding, None }
>
> @penev92 <https://github.com/penev92> In RA it can take out a conyard. In
> D2k it definitely can't. Dunno TD.
>
> @fadyanwar <https://github.com/fadyanwar> I was a really good mood.
> http://pastebin.com/i39ndBXJ Thank you for ruining it with a spoonfeeding
> request about something which enough people told you already.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenRA/OpenRA/pull/9936/files#r44680079>.
>



-- 
Best Regards,
Fady Anwar